redash/query_runner/mysql.py

```diff
@@ -46,9 +46,25 @@ def configuration_schema(cls):
                     'type': 'string',
                     'title': 'Database name'
                 },
-                "port": {
-                    "type": "number"
+                'port': {
+                    'type': 'number'
                 },
+                'use_ssl': {
+                    'type': 'boolean',
+                    'title': 'Use SSL'
+                },
+                'ssl_cacert': {
+                    'type': 'string',
+                    'title': 'CA certificate to verify peer against (SSL)'
```

Comment:
If we stored the keys in the DB, anyone with redash DB access would have access to private, unencrypted keys stored in the DB, which may not be in line with an organization's security policies. Having the keys stored on the filesystem, which may be encrypted at rest, at least gives a system admin the ability to restrict access using filesystem permissions.